zip-codes/scripts/search.js

```diff
@@ -1,9 +1,45 @@
 'use strict';
 
 (function(module) {
-
+  webDB.execute(
+    'SELECT DISTINCT state FROM zips ' +
+    'ORDER BY state DESC;',
```

Comment:
Currently, they are sorted backwards. I think you might want to specify ASC instead of DESC.